<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnHome">
        <property name="text">
         <string>Home</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #2ECC71; /* Verde vivo */
              color: white;
              border: none;
              border-radius: 8px;
              padding: 8px 16px;
              font-weight: bold;
              margin: 4px;
          }
          QPushButton:hover {
              background-color: #27AE60; /* Verde un poco más oscuro */
          }
          QPushButton:pressed {
              background-color: #229954; /* Verde aún más oscuro */
          }
          QPushButton:checked {
              background-color: #1ABC9C; /* Turquesa para el botón seleccionado */
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDatos">
        <property name="text">
         <string>Datos</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #2ECC71; /* Verde vivo */
              color: white;
              border: none;
              border-radius: 8px;
              padding: 8px 16px;
              font-weight: bold;
              margin: 4px;
          }
          QPushButton:hover {
              background-color: #27AE60; /* Verde un poco más oscuro */
          }
          QPushButton:pressed {
              background-color: #229954; /* Verde aún más oscuro */
          }
          QPushButton:checked {
              background-color: #1ABC9C; /* Turquesa para el botón seleccionado */
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHistorial">
        <property name="text">
         <string>Historial</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #2ECC71; /* Verde vivo */
              color: white;
              border: none;
              border-radius: 8px;
              padding: 8px 16px;
              font-weight: bold;
              margin: 4px;
          }
          QPushButton:hover {
              background-color: #27AE60; /* Verde un poco más oscuro */
          }
          QPushButton:pressed {
              background-color: #229954; /* Verde aún más oscuro */
          }
          QPushButton:checked {
              background-color: #1ABC9C; /* Turquesa para el botón seleccionado */
          }
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>